app/src/main/res/values/strings.xml

```diff
@@ -9,7 +9,7 @@
     <string name="ui_item_contact_hint_call">Call</string>
 
     <string name="title_ui">User Interface</string>
-    <string name="title_history">History</string>
+    <string name="title_history">History settings</string>
```

Comment:
Please use an upper-case `S` to be consistent with the title-case used for all the other sections